packages/mongodb-runner/README.md

```diff
@@ -0,0 +1,68 @@
+# mongodb-runner
+
+Helper for spinning up MongoDB servers and clusters for testing.
+
+## Example usage
+
+> Note: Version 5 of mongodb-runner is a full re-write. Many things work
+> differently in previous versions.
+
+```bash
+$ npx mongodb-runner start -t sharded
+$ npx mongodb-runner start -t replset -- --port 27017
+$ npx mongodb-runner start -t replset -- --setParameter allowDiskUseByDefault=true
+$ npx mongodb-runner stop --all
+$ npx mongodb-runner exec -t standalone -- sh -c 'mongosh $MONGODB_URI'
+$ npx mongodb-runner exec -t standalone -- --setParameter allowDiskUseByDefault=true -- sh -c 'mongosh $MONGODB_URI'
+```
+
+## Options
+
+```
+mongodb-runner <command>
+
+Commands:
+  mongodb-runner start  Start a MongoDB instance
+  mongodb-runner stop   Stop a MongoDB instance
+  mongodb-runner ls     List currently running MongoDB instances
+  mongodb-runner exec   Run a process with a MongoDB instance (as MONGODB_URI
+                        env var)
+
+Options:
+  -t, --topology     Select a topology type
+           [choices: "standalone", "replset", "sharded"] [default: "standalone"]
+      --arbiters     number of arbiter nodes for each replica set       [number]
```

Comment:
Minor thing for all these numbers: Should we specify the default? Is there even a default in every case? I suppose each one only matters for specific code paths.

Reply:
So, just for context, the `default:` in yargs sets the actual default, it’s not just informative for the user … so setting e.g. `default: 2` for `secondaries` means that we’d maintain the default in two places. That might be a bit nicer for UX, but I’m not sure if it’s really worth it?

Reply:
Yeah we're the only users, so probably not worth it.